JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -35,7 +43,54 @@ FlowSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that is use
+Screen,Class,ComputerPart,IS-A,,,,A device on which a computer interface is projected onto.,https://www.theworldavatar.com/kg/ontodevice/,Screen
+Projector,Class,ComputerPart,IS-A,,,,A device tht projects an image to a screen,https://www.theworldavatar.com/kg/ontodevice/,Projector
+Speaker,Class,ComputerPart,IS-A,,,,A device that converts electrical signal from  acomputer to soundwaves.,https://www.theworldavatar.com/kg/ontodevice/,Speaker
+Scanner,Class,ComputerPart,IS-A,,,,A device that  scans paper-based documents to a digital format,https://www.theworldavatar.com/kg/ontodevice/,Scanner
+Dongle,Class,ComputerPart,IS-A,,,,"A small piece of computer hardware that connects to a port on another device to provide it with additional functionality, or enable a pass-through to such a device that adds functionality.",https://www.theworldavatar.com/kg/ontodevice/,Dongle
+ServerChassis,Class,ComputerPart,IS-A,,,,A device where servers are mounted on.,https://www.theworldavatar.com/kg/ontodevice/,Server chassis
+NetworkVideoRecorder,Class,ComputerPart,IS-A,,,,A device that converts video recording in a camera network to a computer-storable format,https://www.theworldavatar.com/kg/ontodevice/,Network video recorder
+NetworkDevice,Class,https://saref.etsi.org/core/Device,IS-A,,,,A device that allows communication within a computer network,https://www.theworldavatar.com/kg/ontodevice/,Network devices
+Router,Class,NetworkDevice,IS-A,,,,A device that forwards data packets in a coputer network.,https://www.theworldavatar.com/kg/ontodevice/,Router
+NetworkSwitch,Class,NetworkDevice,IS-A,,,,A device that forwards and receive data from a computer network to a single specific device.,https://www.theworldavatar.com/kg/ontodevice/,Network switch
+Controller,Class,https://saref.etsi.org/core/Device,IS-A,,,,A device that controls a certain variable of its environment.,https://www.theworldavatar.com/kg/ontodevice/,Controller
+FlowController,Class,Controller,IS-A,,,,A device that manages the amount of gas or liquid flowing through.,https://www.theworldavatar.com/kg/ontodevice/,Flow controller
+MassFlowController,Class,FlowController,IS-A,,,,A device that manages the mass of gas or liquid flowing through.,https://www.theworldavatar.com/kg/ontodevice/,Mass flow controller
+Valve,Class,FlowController,IS-A,,,,A device that manages the amount of gas or liquid flowing through via changing the area the material can pass through.,https://www.theworldavatar.com/kg/ontodevice/,Valve
+SelectorValve,Class,Valve,IS-A,,,,A valve that selects what gas or liquid pass through.,https://www.theworldavatar.com/kg/ontodevice/,Selector valve
+FurnaceController,Class,Controller,IS-A,,,,A device that manages the operations of a furnace.,https://www.theworldavatar.com/kg/ontodevice/,Furnace controller
+IlluminationController,Class,Controller,IS-A,,,,A device that manages the level of light a device produces.,https://www.theworldavatar.com/kg/ontodevice/,Illuminance controller
```

Comment:
Usually the label will follow the concept's name. IlluminationController -> "Illumination controller".